docs/pages/eas-update/getting-started.mdx

```diff
@@ -84,18 +84,16 @@ The `channel` allows you to point updates at builds. For example, if we set up a
 <Step label="4">
 ## Create a build for the project
 
-Next, we'll need to create a build for Android or iOS. [Learn more](/build/setup).
+You need to create a build for Android and iOS. We recommend creating a build with the `preview` build profile first. See [Creating your first build](https://docs.expo.dev/build/setup/) on how to get started and set up [Internal distribution](https://docs.expo.dev/build/internal-distribution/#setting-up-internal-distribution) for your device or simulator.
```

Comment:
Why is it mandatory to create a build for both Android and iOS? Think I prefer keeping the `or`.

Reply:
Agreed